test/fixtures/unix.js

```diff
@@ -6271,11 +6271,11 @@ export const quote = {
         expected: "'a b c'",
       },
       {
-        input: "a\r\n",
+        input: "a ",
         expected: "'a '",
       },
       {
-        input: "\r\na",
+        input: " a",
```

Comment:
This change is unintended and should be reverted

```suggestion
        input: "\r\na",
```